Revised object setup README and secrets/details files

keshavattrey opened the pull request:
Fixed some typos in pureadmin command to get the api token and a few other instructions

keshavattrey committed:
Fix a few typos in the readme

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/PureStorage-OpenConnect/ansible-playbook-examples?pullRequest=31) <br/>All committers have signed the CLA.

keshavattrey committed:
Revised object setup README and secrets/details files

Reorganized object README.md file, fb_secrets.yml, and fb_details.yml
for readability.
